src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -200,14 +192,20 @@ void LiveFunctionsDataView::DoSort() {
   bool ascending = sorting_orders_[sorting_column_] == SortingOrder::kAscending;
   std::function<bool(uint64_t a, uint64_t b)> sorter = nullptr;
 
-  const absl::flat_hash_map<uint64_t, FunctionInfo>& functions = functions_;
-
   switch (sorting_column_) {
     case kColumnSelected:
-      sorter = ORBIT_CUSTOM_FUNC_SORT(app_->IsFunctionSelected);
+      sorter = MakeFunctionSorter(
+          [this](const FunctionInfo& function_info) {
+            return app_->IsFunctionSelected(function_info);
+          },
+          ascending, false);
```

Comment:
Is there any problem with previous sorters? I prefer to not change them. 
Take the Selected column as example, the previous behavior in live functions data view and functions data view are the same. But I'm not sure whether they still keep consistent after the change.

Also the selected column not only shows whether a function is selected or not, but also shows the frame track is enabled or not. And now we are only sorting according to the selection.